server/game/gameSystems/ResourceCardSystem.ts

```diff
@@ -28,15 +29,21 @@ export class ResourceCardSystem<TContext extends AbilityContext = AbilityContext
         // TODO: remove this completely if determinmed we don't need card snapshots
         // event.cardStateWhenMoved = card.createSnapshot();
         const properties = this.generatePropertiesFromContext(context, additionalProperties) as IResourceCardProperties;
-        //TODO: Is there a better/cleaner way to handle one or multiple cards here?
-        const cards = [].concat(properties.target);
-        cards.forEach((card) => {
-            const player = properties.changePlayer && card.controller.opponent ? card.controller.opponent : card.controller;
-            player.moveCard(card, Location.Resource);
-            if (properties.readyResource) {
-                context.game.openEventWindow(ready({ target: card }).generateEvent(context.source, context));
-            }
-        });
+        // TODO: If we ever need to resource multiple cards at once, this will need an update
+        const card = Array.isArray(properties.target) ? properties.target[0] as Card : properties.target as Card;
+
+        const player = properties.targetPlayer === RelativePlayer.Opponent ? card.controller.opponent : card.controller;
```

Comment:
Not for this PR, but this kind of makes me wonder if there needs to a be `WildcardRelativePlayer` type that just has `WildcardRelativePlayer.Any`. Because `RelativePlayer.Any` is technically a legal value here but it makes no sense for it to be that.

For now would be great to add a contract check in the overridden `generatePropertiesFromContext` for that and a TODO to investigate further.